src/DotVVM.Framework/Controls/GridViewColumn.cs

```diff
@@ -127,6 +128,47 @@ public bool Visible
         public static readonly DotvvmProperty VisibleProperty
             = DotvvmProperty.Register<bool, GridViewColumn>(c => c.Visible, true);
 
+        /// <summary>
+        /// Gets or sets a list of decorators that will be applied on each cell which is not in the edit mode.
+        /// </summary>
+        [PopDataContextManipulation]
```

Comment:
Cell decorators shouldn't pop, they are in the row context.

Reply:
Good catch, thanks!